### PR TITLE
Pract.04: Add 04-06 | Momchil Uzunov

### PR DESCRIPTION
**Pull request by momouzunov2003**
Momchil Uzunov
6 1MI0600146

### CODE REVIEW

File: SI/Sem.04/Pract.04/Task 06.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    else
+    {
+        return false;
+    }
+}
+
+bool checkSymbol(char S)
+{
+    if ((S >= 'A' && S <= 'Z') || (S>='a' && S<='z'))
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 06.cpp:17]: (S >= 'A' && S <= 'Z') || (S>='a' && S<='z') is a boolean expression, so it returns a boolean value that you can directly return

File: SI/Sem.04/Pract.04/Task 04.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+
+int abs(int num)
+{
+    if (num >= 0)
+    {
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 04.cpp:6]: You can use a ternary operator here

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
Some refactoring needed

**Comment by momouzunov2003** [comment]
Updated tasks 04 and 06.

File: SI/Sem.04/Pract.04/Task 04.cpp [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+int abs(int num)
+{
+    int result=(num >= 0) ? num :  -num;
+    return result;
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 04.cpp:6]: No need of a variable here. You can directly return

File: SI/Sem.04/Pract.04/Task 04.cpp [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+double fabs(double num)
+{
+   int result = (num >= DBL_EPSILON) ? num : -num;
+    return result;
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 04.cpp:12]: Not working correctly. You are assigning a double to int so you are returning the int that will be converted to double

File: SI/Sem.04/Pract.04/Task 06.cpp [hunk 5]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+bool isCapitalLetter(char S)
+{
+    return (S >= 'A' && S <= 'Z');
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 06.cpp:5]: No need of brackets

File: SI/Sem.04/Pract.04/Task 06.cpp [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+bool checkSymbol(char S)
+{
+    return ((S >= 'A' && S <= 'Z') || (S >= 'a' && S <= 'z'));
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/Task 06.cpp:10]: No need of the first brackets

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**